[2023-09-03T09:52:35.435+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim permission to 509
[2023-09-03T09:52:35.436+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim'
[2023-09-03T09:52:35.475+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim permission to 509
[2023-09-03T09:52:35.476+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim'
[2023-09-03T09:52:35.502+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.insert_into_location_dim manual__2023-09-03T09:51:48.060769+00:00 [queued]>
[2023-09-03T09:52:35.511+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.insert_into_location_dim manual__2023-09-03T09:51:48.060769+00:00 [queued]>
[2023-09-03T09:52:35.512+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-09-03T09:52:35.527+0000] {taskinstance.py:1382} INFO - Executing <Task(BigQueryExecuteQueryOperator): insert_into_location_dim> on 2023-09-03 09:51:48.060769+00:00
[2023-09-03T09:52:35.534+0000] {standard_task_runner.py:57} INFO - Started process 4598 to run task
[2023-09-03T09:52:35.536+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Autotrader_ETL_DAG', 'insert_into_location_dim', 'manual__2023-09-03T09:51:48.060769+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/autotrader_etl.py', '--cfg-path', '/tmp/tmpzvg86lhz']
[2023-09-03T09:52:35.539+0000] {standard_task_runner.py:85} INFO - Job 316: Subtask insert_into_location_dim
[2023-09-03T09:52:35.588+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim permission to 509
[2023-09-03T09:52:35.589+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-03T09:51:48.060769+00:00/task_id=insert_into_location_dim'
[2023-09-03T09:52:35.592+0000] {task_command.py:415} INFO - Running <TaskInstance: Autotrader_ETL_DAG.insert_into_location_dim manual__2023-09-03T09:51:48.060769+00:00 [running]> on host 1a0e8b05e04d
[2023-09-03T09:52:35.687+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Autotrader_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_into_location_dim' AIRFLOW_CTX_EXECUTION_DATE='2023-09-03T09:51:48.060769+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-03T09:51:48.060769+00:00'
[2023-09-03T09:52:35.691+0000] {bigquery.py:1234} INFO - Executing: INSERT INTO autotrader-toyota-dashboard.autotrader_transformed.location_dim (location_key, suburb, state)
        SELECT
        ROW_NUMBER() OVER () as location_key,
        suburb,
        state
        FROM (
        SELECT 
        DISTINCT
        suburb,
        state
        FROM
        autotrader-toyota-dashboard.autotrader_staging.listings_raw
        );
[2023-09-03T09:52:35.700+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2023-09-03T09:52:35.704+0000] {base.py:73} INFO - Using connection ID 'google_cloud' for task execution.
[2023-09-03T09:52:35.710+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/google/cloud/hooks/bigquery.py:2109 AirflowProviderDeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2023-09-03T09:52:35.755+0000] {bigquery.py:1596} INFO - Inserting job ***_1693734755755150_945dcafba0ba8e574853943552db7aa1
[2023-09-03T09:52:38.668+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Autotrader_ETL_DAG, task_id=insert_into_location_dim, execution_date=20230903T095148, start_date=20230903T095235, end_date=20230903T095238
[2023-09-03T09:52:38.722+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-03T09:52:38.750+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
